### DISCUSSION TITLE
Media Recorder issues

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#183] [! Welcome - readme first!]
Hey Cables massive.

Lately I have been trying to get some HD 60fps recordings out of my patches, but am having issues.  Pretty much everytime I try to render, the resulting video file has a broken duration which means it won't play in VLC or other stuff.  

I can fix it sometimes with the awesome https://github.com/mifi/lossless-cut  

I am on Mac on Chrome. Not sure how to create proper bug report with this...

**Comment by pandrr** [comment]
hi,

can you test this on dev.cables.gl ? we improved the op there.

**Reply by cristianvogel** [reply]
It works much better in terms of the quality of image, but the duration corruption is still there and currently means that it is not possible to change formats for editing in DaVinci for example. The dev.cables outputs also don't seem to be fixable by LosslessCut. 

I tried 18mbit 60fps MKV and a few others

**Comment by tastez** [comment]
MediaRecorder API doesn't add file duration on creation of the recording. Here's the issue and solution is to fix duration manually, depending on your use case. I think your solution is very valid here @cristianvogel so my advice is everyone do that if they need to reimport the video, or use another tool to prep their video recordings (like HandBrake which i tested and it allowed DaVinci import). In simple use cases like embedding or sharing screengrabs the duration and seeking the file isn't needed and most social media platforms autoconvert and add the duration/seekbar.
 
https://bugs.chromium.org/p/chromium/issues/detail?id=642012
